packages/prisma-client/.gitignore

```diff
@@ -0,0 +1,2 @@
+lib
+generated
```

Comment:
both should probably be or maybe already are globally ignored, because they sound like every package could use them

Reply:
Done!